src/Extensions/Hosting.Services.Remoting/Remoting.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.ServiceFabric.Services.Remoting.FabricTransport.Runtime;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Remoting
+{
+	/// <summary>
+	/// Static methods to simplify creation of <see cref="FabricTransportRemotingListenerSettings" />
+	/// </summary>
+	public static class Remoting
+	{
+		/// <summary>
+		/// Creates <see cref="FabricTransportRemotingListenerSettings" /> with provided EndpointResourceName
+		/// </summary>
+		public static FabricTransportRemotingListenerSettings EndpointResourceName(string value)
```

Comment:
>value [](start = 84, length = 5)

You could make this an extension method. Similar designs have been used for similar code in the past, although the use of the `string` type here means this feels a little less beneficial that for more esoteric types

Reply:
Yes I feel like `string` is too generic and with new name, it should be better as regular static method